### PR TITLE
[#11150] Search page: Remove unnecessary panel 

### LINKED ISSUE #11150: Search page: Remove unnecessary panel
image

Suggested changes:
* Rephrase as `You can search for students (by name or email) from all your courses here.`
* Give an example at the end `e.g., "John Doe"`
* Remove this outer panel as all results are students anyway.

### PR DESCRIPTION
**Pull request by tamrakarsweta220**
Fixes #11150

**PR Checklist**

Ensure that you have:
- [x] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the above mentioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [x] Added/updated tests, if changes in functionality were involved
- [x] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

I made two changes to the instructions on the student search page. 
1. Updated the search instruction to let users know that they can search by name or email
2. Gave an example for using quotations in the search bar to search for specific phrases at the end of the instruction.

I also removed the "student" header and the outer panel from the search result table. 

This is how the changes currently look like: 
Screen Shot 2022-03-27 at 8 59 59 PM

### CODE REVIEW

**Review by FergusMok — COMMENTED** [review]
Hello, thank you for contributing to TEAMMATES. It seems like some linting tests are failing too

You can refer to the [documentation for linting ](https://github.com/TEAMMATES/teammates/blob/master/docs/static-analysis.md). To update the snapshots, you can run `npm run test -- --updateSnapshot`, or alternatively, run `npm run test`, and then press the button "u".

**Comment by fsgmhoward** [comment]
From your screenshot, it looks like all your tests are failing. Have you execute this before you run the test:
```
./gradlew generateTypes
```

If that is the case, you will need to execute this before you run the test. This command generates frontend types from backend description files. After that, you can run test with and update snapshot based on displayed information (see @FergusMok 's comment above).
```
npm run test
```

Even though the change is minor, it is still suggested to follow steps in [Development Guide](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md) to run the development server and test the functionality.

**Comment by tamrakarsweta220** [comment]
I followed the instructions as suggested above by @fsgmhoward and @FergusMok and updated the snapshot, so the component tests have passed. I am still having issues with lint though. I tried running the `npm run lint` command for the master branch too, but it's the same. The lint checks fail in the master branch too.

Also, the teammates bot keeps telling me that my PR "Should contain GitHub keyword to auto-close issue it fixes: Refer [here](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) for a list of accepted keywords." I wrote "fixes" based on the documentation but it's still showing me this error. I capitalized the "f" in "fixes" and then put it in lowercase too, but the error didn't go away. So, I am not sure what is causing the error. Is there something else that I am missing?

**Comment by fsgmhoward** [comment]
> 



> I followed the instructions as suggested above by @fsgmhoward and @FergusMok and updated the snapshot, so the component tests have passed. I am still having issues with lint though. I tried running the `npm run lint` command for the master branch too, but it's the same. The lint checks fail in the master branch too.
> 
> Also, the teammates bot keeps telling me that my PR "Should contain GitHub keyword to auto-close issue it fixes: Refer [here](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) for a list of accepted keywords." I wrote "fixes" based on the documentation but it's still showing me this error. I capitalized the "f" in "fixes" and then put it in lowercase too, but the error didn't go away. So, I am not sure what is causing the error. Is there something else that I am missing?

Usually we do with "Fixes". I am not sure whether it is accepted or not in lower case.

For the lint issue, I see that all checks passed already. It fails on your local machine because the lint program takes auto-generated files, which will fail it. You have to take a look at which file fails the linting test. If it is not those you made changes, then you are fine. If you are interested, the related issue is this: #11564

File: src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <div class="card bg-light mb-4">
   <div class="card-body">
     <div class="text-muted">
-      You can search for students from all your courses here.
+      You can search for students (by name or email) from all your courses here.
```
Comment (FergusMok) [inline src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.html:4]: The search bar can actually search for more fields according to the Help page. i.e. Section name, Team name. I know the issue suggested name and email, but would it be better if we could indicate all the fields to be searched?

**Review by FergusMok — CHANGES_REQUESTED** [review]
Good job, the table looks good. Just some small considerations, and remember to update the branch to master

File: src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.html (hunk 1 above)
Comment (tamrakarsweta220) [reply src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.html]: Update: I added all fields (section name, team name, student name, and email) that can be used to search in the search bar and pushed those changes. Now, I am waiting to see if all checks pass.

**Comment by tamrakarsweta220** [comment]
@FergusMok Hi, I just made the changes you suggested. So, I'm commenting for code review again. Thanks!

**Review by fsgmhoward — APPROVED** [review]
LGTM.

**Review by FergusMok — APPROVED** [review]
LGTM! 👍

**Pull request merged**